lib/App/Asciio/Actions/Mouse.pm

```diff
@@ -375,7 +375,7 @@ my ($self, $event) = @_ ;
 
 my ($x, $y) = @{$event->{COORDINATES}}[0, 1] ;
 
-if($event->{STATE} eq 'dragging-button1' && ($self->{PREVIOUS_X} != $x || $self->{PREVIOUS_Y} != $y))
+if($event->{STATE} eq 'dragging-button3' && ($self->{PREVIOUS_X} != $x || $self->{PREVIOUS_Y} != $y))
 	{
 	my ($character_width, $character_height) = $self->get_character_size() ;
```

Comment:
where is it documented that button-3 is the one dragging canvas?

most functions have shortcuts and are visible in the default bindings but these kind of magic must be clearly documented or it should be via a binding that is defined in the bindings file